.github/CONTRIBUTING.md

```diff
@@ -77,24 +77,106 @@ Pull Requests complies with these rules:
+    file][JSDoc config]
 
 ## Testing
 
-Before submitting a PR please verify that your code passes all the tests. To do
-that either create a new script-file for `unit-tests.gs` in the Google
-script-editor, or append the content of `unit-tests.gs` to that of `code.gs` in
-its own file, then `run->unitTests()`. It is good to also `run->test()` with
-`settings.debug.testDate` set to a date with some contact-anniversaries on it
-(or create some fake ones on that date) to provide real-world testing too. For
-exhaustive real-world testing there is also `testSelectedPeriod()`. If there are
-any other new global functions in the [tests file][Tests file] you can run them
-too.
+Before submitting a PR:
+
+- If you've updated translation strings, please check that you have used the same
+  format as the other entries, for example:
+  ```javascript
+  'You can find the latest one here': 'Puoi trovare l\'ultima qui',
+  ```
+  - Both the "from" and "to" parts of each entry are surrounded by single-quotes
+    (`'aa'`)
+  - Any single-quotes within the "from" or "to" are escaped with a backslash
+    (`'a\'a'`)
+  - There is a colon and space between the "from" and "to" (`'aa': 'bb'`)
+  - Each entry has a trailing comma (`... : 'bb',`)
+- If you've made updates to the javascript code in any `.gs` files:
+  - To lint for syntax errors [this semistandard linter][Javascript semistandard]
+    is one of the options available as a plugin for many editors and can also be
+    run as a commandline tool using (for example, on a Unix-like system):
+    ```sh
+    npm install -g "semistandard" "snazzy" # if not yet installed
+    semistandard --verbose `find . -name "*.gs"` | snazzy
+    # "snazzy" is an optional pretty-printer
+    ```
+  - To check for semantic errors:
+    - Please verify that your code passes all the tests. To do
+      that either create a new script-file for `tests.gs` within the same
+      project in the Google script-editor, or append the content of
+      `tests.gs` to that of the `code.gs` script-file, then `run->unitTests()`.
+    - It is good to also `run->test()` with `settings.debug.testDate` set to a date
+      with some contact-anniversaries on it (or create some fake ones on that date)
+      to provide real-world testing too.
+    - For exhaustive real-world testing there is also `testSelectedPeriod()`. Beware
+      that this test *might* hit an execution timeout limit.
+    - If there are any other new global functions in the [tests file][Tests file]
+      you can run them too.
+- If you've made updates to any Markdown files:
+  - To lint for syntax errors [this Markdown linter][Markdown linter] is one of the
+    options available as a plugin for many editors and can also be run from a
+    commandline tool using (for example, on a Unix-like system):
+    ```sh
+    npm install -g "markdownlint-cli" # if not yet installed
+    find . -name "*.md" -exec markdownlint {} \;
```

Comment:
I'd add `-c ".markdownlint.json"` here.

Reply:
Oops, forgot the flag. Will add, thanks.